.storybook/config.js

```diff
@@ -41,6 +41,9 @@ const stories = [
   // CodeViewer
   require.context("./components/CodeViewer", false, /CodeViewer.story.js/),
 
+  // Card
```

Comment:
@mikrotron why do we have to do this? This is usually automated

Reply:
Gives us control over the ordering.  Probably a better way, this is just a temporary workaround.

Reply:
yes, I have on my list to remove this, and dynamically load it and sort it by a json or something else.